### ISSUE TITLE
Better README?

### ISSUE THREAD

**Issue by hbredin** [clement-pages/gryannote#22]
Would be nice to add a README at the root of the directory with screenshots and a few instructions on how to use gryannote.

**Comment by clement-pages** [comment]
Good idea, I will integrate it in #20. This PR already add a basic `README` at the root.

**Comment by clement-pages** [comment]
All the `README` files have been updated with screenshots and some instructions for each component.
